notebooks/Notebook2.md

```diff
@@ -18,17 +31,49 @@ exciting milestones, preliminary results, etc. Feel free to include images
 code, and any other resources that you think will help clearly convey your
 design process.
 
+I spent a good chunk of this week adapting my Python code into Scala, adding
+some richness to it that takes advantage of Scala's DSL-building features. I
+got pretty stuck while trying to implement a "do for all players" function that
+has a more readable syntax than just using a for loop. The problem is that it's
```

Comment:
I think "time" and spontaneity will be a problem that will poke its head a lot in your project. I recommend really defining how it will work before going forward. Not that I have a great recommendation lol